free-1.md

```diff
@@ -1,54 +1,76 @@
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+One reason why I see a DSL as a potential solution is because current music software includes a lot of other things. Many programming languages, libraries, and APIs that implement ways to create music focus on making it possible to make cool sounds and patterns. But instead of having that affect one's compositions (following the Sapir-Whorf hypothesis), having a programming langauge specific to enhancing one's compositional creativity would decrease the impact of languages on musician's compositions. A DSL can be used as a preliminary tool or a tool in conjunction with other music software. As stated above, the end goal is to let users easily think of problems other than the structure of a composition that arise when composing.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am interested in music in different ways. I take piano lessons, do chamber music, play gamelan, play jazz, and am a part of the Pomona College Orchestra. So music (at least classical and jazz) has been a large part of my life, starting when I was fairly young.
+
+I took an independent study in composition with Prof. Alves two years ago, and I learned a couple of things. One is that all pieces, regardless of their genre, always had a plan behind them. Regardless of whether randomness is involved, I believe that almost all music depends on structure and organization. The structure doesn't have to be rigid. For example, a structure can be just 3 sections that have completely random notes, rhythms, and dynamics. However, humans tend to enjoy music they understand, and having structure helps them understand the music.
+
+Overall, this idea is inspired by my struggles in composition, and my hope that composing can be easier for everyone.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Music Composition Organization
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I'm not sure how the user would interact with the language. It might be have a gui where people can edit a flow chart. It might also just be something like a regular language. However, I'd like to make sure that the user is able to clearly see the organization.
```

Comment:
What are some advantages and disadvantages of each method? If you write them out and reason about them that will help push you in the right direction.